### DISCUSSION TITLE
Challenger RP2040 WiFi

### DISCUSSION THREAD

**Discussion by Buddha56** [earlephilhower/arduino-pico#1293] [Q&A] [answered]
Hello Community,
Is there a way to use the full scope of the WIFI of this LIB (WIFI 8266).  I would like to use the update functions. 

If so how ?
I would like to use the Arduino IDE.

**Comment by JAndrassy** [comment, marked as answer]
see the Examples tab here
https://ilabs.se/product/challenger-2040-wifi-chip/

**Reply by Buddha56** [reply]
Thank you . 
The WIFI works. The only thing I would like to do now is to implement the HTTP update function. As soon as I integrate the HTTPupdate.h, it "links" directly to the WIFI of the PICOW. Is there an "easy" way to bypass this and use libraries that support the ESP8266 ?

**Comment by PontusO** [comment]
I did some light checking and it looks like this lib unfortunately also ties into the rest of the WiFi structure that is more or less hardcoded to the Pico hardware.

You could check out the [ArduinoOTA](https://github.com/JAndrassy/ArduinoOTA) library from @JAndrassy which works very well with the Challenger WiFi based boards.
